great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -1,8 +1,12 @@
+import configparser
+import logging
+import os
 from abc import ABC, abstractmethod
+from typing import Optional
 
-from great_expectations.data_context.types.data_context_variables import (
-    DataContextVariables,
-)
```

Comment:
In the current state of the PR, that's correct. I believe this will have to be re-introduced to `AbstractDataContext` as we continue to move code up from `BaseDataContext`.